core/controllers/base_test.py

```diff
@@ -107,9 +120,55 @@ def test_redirect_in_both_logged_in_and_logged_out_states(self):
         self.login('user@example.com')
         response = self.testapp.get('/')
         self.assertEqual(response.status_int, 302)
+
+        # Homepage is either dashboard or library based on user's learner
+        # status, this homepage redirect rule behavior is seperately tested
+        atHome = ('dashboard' in response.headers['location'] or
```

Comment:
@jaredsilver To your last point, I'd say no, not really. At most, they can give suggestions, but the eventual "author" for those is the person who actually merges it into the exploration. So what you propose in the first paragraph is fine.